rpc/pano-api/service.proto

```diff
@@ -0,0 +1,189 @@
+syntax = "proto3";
```

Comment:
we shouldn't need these files in this repo, these should be imported from the external package